src/test/java/io/dashboardhub/pipelinedashboard/definitions/ProjectDefinitions.java

```diff
@@ -10,6 +10,8 @@
 import org.junit.Assert;
 import org.springframework.beans.factory.annotation.Autowired;
 
+import java.util.UUID;
```

Comment:
I would remove for now, to keep it cleaner. We can add back into the PR that requires them